sql/database.sql

```diff
@@ -654,6 +654,15 @@ CREATE TABLE `clinical_rules` (
   `web_reference` VARCHAR(255) NOT NULL DEFAULT '' COMMENT 'Clinical Rule Web Reference',
```

Comment:
this needs all the other stuff from changes in 7_0_2-to-7_0_3_upgrade.sql

Reply:
fixed